examples/roitberg/pyanitools.py

```diff
@@ -0,0 +1,126 @@
+# Written by Roman Zubatyuk and Justin S. Smith
```

Comment:
I haven't decided if we should bundle this or not in the long run, but for now I'm including it because it makes dependency management easier.